docs/stock/hollow-candlestick-chart.md

```diff
@@ -0,0 +1,16 @@
+Hollow Candlestick chart
+================
+
+A hollow candlestick series is similar to a candlestick series, with a few modifications. The hollow candlestick point is composed of a body and an upper and a lower wick, however, it uses color and fill attributes to show price behavior. 
+
+
+If the current closing price is higher than the opening price (bullish candle) the body is transparent. <br>
+When the current close is lower than the previous close, the line and filling for bearish candles has a reddish color.<br> 
```

Comment:
Are those `<br>` necessary in the .md files? Simple `enter` should be enough (I am sure)